keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/kep.yaml

```diff
@@ -31,17 +35,18 @@ latest-milestone: "v1.19"
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.19"
-  stable: "v1.20"
+  beta: "v1.20"
+  stable: "v1.21"
```

Comment:
I suggest 1.22 to really have time for gathering feedback in Beta.

Reply:
ok